cypress/e2e/HomePage/DeleteProject.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Test home page: project deletion
```

Comment:
All the added tests files have been deleted in my E2E pr.
I think you did not rebase correctly